### ISSUE TITLE
[LiveComponent] Cancel polling

### ISSUE THREAD

**Issue by easis** [symfony/ux#382]
Hi,
I have a component with `data-polling` because I need to wait for data but once I have the data there is no way to stop/cancel polling. I've tried conditionally printing the `data-polling` attr with a prop like this with no success:
```twig
<div {{ attributes }} {{ not this.isCompleted? 'data-polling="delay(1000)|check"' }}>
    ...
</div>
```

**Comment by weaverryan** [comment]
fixed in #385 :)

**Comment by easis** [comment]
Thank you!
